src/ipa_pc/mod.rs

```diff
@@ -641,16 +817,19 @@ impl<G: AffineCurve, D: Digest> PolynomialCommitment<G::ScalarField> for InnerPr
         labeled_polynomials: impl IntoIterator<Item = &'a LabeledPolynomial<G::ScalarField>>,
         commitments: impl IntoIterator<Item = &'a LabeledCommitment<Self::Commitment>>,
         point: G::ScalarField,
-        rands: impl IntoIterator<Item = &'a LabeledRandomness<Self::Randomness>>,
-        rng: Option<&mut dyn RngCore>,
         // This implementation assumes that commitments, query point and evaluations are already absorbed by the Fiat Shamir rng
         fs_rng: &mut Self::RandomOracle,
+        rands: impl IntoIterator<Item = &'a LabeledRandomness<Self::Randomness>>,
+        rng: Option<&mut dyn RngCore>,
```

Comment:
I would have kept with the previous order of the function arguments.  
First, everything that the interactive protocol needs (including the blinding rands and rng), then the Fiat-Shamir.